### ISSUE TITLE
Mouse & Keyboard

### ISSUE THREAD

**Issue by Daandelange** [jvcleave/ofxRPI4Window#9]
Hey Jason,
I've investigated enabling the mouse and keyboard functionality. Do you have any ideas/directions yet ?

One candidate I found is [RayLib](https://github.com/raysan5/raylib) which could serve as a wrapper for launching OF.
It could also handle window creation (shared code = more maintained?) in a similar way you already do.
[This SO discussion](https://stackoverflow.com/questions/25557027/linux-keyboard-raw-reading-whats-better-reading-from-dev-input-event0-or-rea) from the author about grabbing mouse/keyboard without desktop environment. It states using `stdin` is limited for games (and thus a lot of ofApps?), so he [ended up](https://github.com/raysan5/raylib/blob/6f3b2a76614369f03721b9bac71727b6a5fcf393/src/rcore.c#L670-L673) using [libEvDev](https://linux.die.net/man/4/evdev), which looks quite complicated, but not more then using the DRM api.

**Comment by jvcleave** [comment]
I haven't done any work in this area. Raylib seems interesting and could merit it's own addon?

**Comment by Daandelange** [comment]
Yes, probably, it could even be an alternative oF launcher with some work.
